include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -968,4 +968,18 @@ def ProbesToSignals : Pass<"firrtl-probes-to-signals", "firrtl::CircuitOp"> {
   let constructor = "circt::firrtl::createProbesToSignalsPass()";
 }
 
+def CheckRecursiveInstantiation : Pass<"firrtl-check-recursive-instantiation",
+    "firrtl::CircuitOp"> {
+  let summary = "Check for illegal recursive instantiation";
+  let description = [{
+    This pass checks for illegal recursive module instantion. Recursive
```

Comment:
```suggestion
    This pass checks for illegal recursive module instantion.  Recursive
```
(I don't know what we standardize on, but later it's two spaces so went with that, just as long as it's consistent at least in same paragraph!)